py3status/modules/window_title.py

```diff
@@ -41,27 +42,27 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 0.5
+    format = '{title}'
     max_width = 120
 
     def __init__(self):
-        self.text = ''
+        self.title = ''
 
     def window_title(self):
         window = find_focused(i3.get_tree())
 
         transformed = False
-        if window and 'name' in window and window['name'] != self.text:
-            self.text = (len(window['name']) > self.max_width and
+        if window and 'name' in window and window['name'] != self.title:
+            self.title = (len(window['name']) > self.max_width and
                          "..." + window['name'][-(self.max_width - 3):] or
```

Comment:
`u"...{}".format(window['name'][-(self.max_width - 3):]) or`

the whole bit of code here makes me a little sick in my mouth